src/project/view/recentscoresmodel.cpp

```diff
@@ -75,22 +75,27 @@ void RecentScoresModel::updateRecentScores()
 
         std::string suffix = io::suffix(file.path);
         bool isSuffixInteresting = suffix != engraving::MSCZ;
+
+        RetVal<uint64_t> fileSize = fileSystem()->fileSize(file.path);
+        QString fileSizeString = (fileSize.ret && fileSize.val > 0) ? DataFormatter::formatFileSize(fileSize.val).toQString() : QString();
```

Comment:
Don't ignore ret

Reply:
It's not ignored: it's checked in `(fileSize.ret && fileSize.val > 0) ? … : …`. And in my opinion we _really_ should not add any error logging here if the size of a file cannot be determined; that's useless cluttering of log files. This information is very unimportant anyway, so if it cannot be loaded for some scores, nobody cares. It does not have any functional value.